interp/src/interpreter/interpret_control.rs

```diff
@@ -1,34 +1,62 @@
 //! Inteprets a control in a component.
 
-use super::interpret_group::interpret_group;
+use super::interpret_group::{finish_group_interpretation, interpret_group};
 use crate::environment::Environment;
 use calyx::{errors::FutilResult, ir};
 
-/// Helper function to evaluate control
 pub fn interpret_control(
```

Comment:
I should write a comment about this, the only reason it exists is to have special printing behavior when the component is just a single group because it effectively prints right _before_ the group's go shuts off when the done goes high, so that the values held on the wires can be observed. Otherwise the values all go back to zero for the most part.

That said, there's nothing special here and we can decided that the printing output for single groups should be exactly the same as anything else. I suppose the more I think on it, the more I'm in favor of getting rid of this and returning the behavior to the previous output.